src/database/KeyValue.ts

```diff
@@ -77,6 +77,32 @@ export const KeyValue = {
       .then(items => items[0]);
   },
 
+  /**
+   * Search in keys with LIKE operator
+   * @param likeKey - case-insensitive
+   * @param flag - a 2 binary mask. 0b10 for left %, 0b01 for right %, 0b11 for both
```

Comment:
this is TS, not C

Reply:
What about something like this:

```ts
// Names TBD
enum LikeSide {
  StartsWith = 0b10,
  EndsWith = 0b01,
  Contains = 0b11,
}
```

Reply:
Voila, j'ai fait un enum SearchFlag et rajouté des cas de tests pour Exact